sections/interfaces.js

```diff
@@ -368,9 +368,17 @@ export default [
     description: "Returns an array of query strings filled out with the correct values based on bindings, etc. Useful for debugging.",
     children: [
       {
-        type: "runnable",
+        type: "code",
+        language: "js",
         content: `
-          knex.select('*').from('users').where(knex.raw('id = ?', [1])).toString()
+          var toStringQuery = knex.select('*').from('users').where('id', 1).toString();
+          console.log(toStringQuery.toString());
+          
+          // toString() must be called on the variable object to print out query to console.
+          // Outputs:
+
+          select * from "users" where id = 1
```

Comment:
this should be in comment, since its not javascript code

Reply:
`// Outputs: select * from "users" where id = 1`

Reply:
Okay interesting my bad, I had a ton of dependency trouble trying to actually run the documentation on a local node.js server.